graylog2-web-interface/src/views/components/visualizations/GenericPlot.jsx

```diff
@@ -209,6 +209,10 @@ class GenericPlot extends React.Component<Props, State> {
             return chart;
           });
 
+          if (newChartData[0].type === 'pie') {
```

Comment:
Can we also do it inside of the `map` just above? It would make it purer and contained.